subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -399,18 +400,55 @@ static int cmd_proxy_solicit(const struct shell *sh, size_t argc,
 #endif /* CONFIG_BT_MESH_SHELL_GATT_PROXY */
 
 #if defined(CONFIG_BT_MESH_SHELL_PROV)
+static int ascii_decimal_to_le_bytes(const char *ascii_str, uint8_t *out, size_t *out_len)
+{
+	for (const char *p = ascii_str; *p; ++p) {
```

Comment:
I guess argv[1] is always NULL-terminated?
```suggestion
	for (const char *p = ascii_str; *p != '\0'; ++p) {
```
Otherwise this should be:
```suggestion
	for (const char *p = ascii_str; *p != NULL; ++p) {
```